modules/apps/frontend-js/frontend-js-dropdown-support-web/src/main/resources/META-INF/resources/DropdownProvider.js

```diff
@@ -69,6 +71,13 @@ class DropdownProvider {
 			return;
 		}
 
+		document.removeEventListener('mousedown', (event) =>
+			this._maybeClickOutside(event, {menu, trigger})
```

Comment:
I think this won't work, you're trying to remove a listener passing a new function, you need to pass the same one you created

Reply:
For solving this, Could We use EventHandler from `metal-events`?